README.md

```diff
@@ -50,22 +82,18 @@ export default Manifest({
 });
 ```
 
-## 3. Writing functions
+#### Writing functions
 
-On our [next-generation platform](https://api.slack.com/future), you can 
-build **Run On Slack functions**, reusable building blocks of automation 
-that are deployed to Slack and accept inputs, perform some calculations, 
-and provide outputs. 
+On our [next-generation platform](https://api.slack.com/future), you can build **Run On Slack functions**, reusable building blocks of automation that are deployed to Slack and accept inputs, perform some calculations, cand provide outputs. 
 
-Functions can be triggered via Global Shortcut, and we'll be adding support 
-for more function and trigger types in the coming months.
+Functions can be triggered via Global Shortcut, and we'll be adding support for more function and trigger types in the coming months.
```

Comment:
I don't believe this is true anymore, right? They must be link triggers? Since this is likely to change, perhaps we remove this all together.